internal/key/key.go

```diff
@@ -27,7 +27,8 @@ const (
 	// (does not contain 1 and l, to avoid confusion)
 	NameChars = "023456789abcdefghijkmnopqrstuvwxyz"
 	// NameLengthLong represents the number of characters used to create a resource name when --enable-long-names feature flag is used.
```

Comment:
i think the --enable-long-names flag is deprecated anyways. maybe we can remove references to it? and also call the var then `NameLengthMax` ? or are there too many references to it?

Reply:
the --enable-long-names is deprecated and it is enabled by default. So the existing name lengths are all defaulting to NameLengthMax